EasyClangComplete.py

```diff
@@ -78,6 +78,21 @@ def run(self, edit):
             log.debug(" Nothing to clean")
 
 
+class ApplyFixitCommand(sublime_plugin.TextCommand):
+    """Command to apply fixit at cursor position."""
+
+    def run(self, edit):
```

Comment:
Is there a way to access the instance of EasyClangComplete (which is required to get config and completer) from ApplyFixitCommand?